src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4748,6 +4830,23 @@ components:
             validators:
               $ref: "#/components/schemas/ValidatorUptimeCollection"
 
+    ResourceOwnersRequest:
+      allOf:
+      - $ref: "#/components/schemas/CursorLimitMixin"
+      - type: object
+        properties:
+          resource_address:
```

Comment:
I see we haven't got a Global / Parent / Vault aggregation option. I assume it's Globally aggregated?

Reply:
yes it's globally aggregated. do we need to vault aggregate as well? that would make it a bit more complex on database and aggregation side.